packages/flutter/lib/src/rendering/sliver.dart

```diff
@@ -957,6 +994,15 @@ class SliverPhysicalParentData extends ParentData {
   /// top left visible corner of the sliver.
   Offset paintOffset = Offset.zero;
 
+  /// The [crossAxisFlex] factor to use for this sliver child.
+  ///
+  /// If null or zero, the child is inflexible and determines its own size.
+  /// If non-zero, the amount of space the child can occupy in the
```

Comment:
This could be confused by developers. So if a developer sets a flex value to 2 in a custom sliver, will it do this? It only works in the context of SliverCrossAxisGroup, this should be clear about that.